examples/mayavi/mlab/tvtk_in_mayavi.py

```diff
@@ -42,17 +42,17 @@
 # The Property will give the parameters of the material.
 p = tvtk.Property(opacity=0.2, color=(1, 0, 0))
 # The actor is the actually object in the scene.
-sphere_actor = tvtk.Actor(mapper=sphere_mapper, property=p)
+sphere_actor = tvtk.Actor(position=(0, 0, 0), mapper=sphere_mapper, property=p)
 v.scene.add_actor(sphere_actor)
 
-# Create a second sphere
-sphere2 = tvtk.SphereSource(center=(7, 0, 1), radius=0.2)
-sphere_mapper2 = tvtk.PolyDataMapper()
-configure_input_data(sphere_mapper2, sphere2.output)
-sphere2.update()
-p = tvtk.Property(opacity=0.3, color=(1, 0, 0))
-sphere_actor2 = tvtk.Actor(mapper=sphere_mapper2, property=p)
-v.scene.add_actor(sphere_actor2)
+# Create a cylinder
+cylinder = tvtk.CylinderSource(center=(0, 0, 0), radius=0.2, resolution=16)
+cylinder_mapper = tvtk.PolyDataMapper()
+configure_input_data(cylinder_mapper, cylinder.output)
+cylinder.update()
+p = tvtk.Property(opacity=0.3, color=(0, 0, 1))
+cylinder_actor = tvtk.Actor(position=(7, 0, 1), mapper=cylinder_mapper, property=p, orientation=(90, 0, 90))
```

Comment:
Could you please wrap this to 80 cols?

Reply:
Sure thing!

Before the change:
```
$ pylint --max-line-length=80 tvtk_in_mayavi.py 
************* Module tvtk_in_mayavi
tvtk_in_mayavi.py:54:0: C0301: Line too long (108/80) (line-too-long)

------------------------------------------------------------------
Your code has been rated at 9.71/10 (previous run: 9.71/10, +0.00)
```
After the change:
```
$ pylint --max-line-length=80 tvtk_in_mayavi.py 

-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 9.71/10, +0.29)
```